lib/galaxy/webapps/galaxy/services/invocations.py

```diff
@@ -0,0 +1,109 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Tuple,
+)
+
+from pydantic import (
+    BaseModel,
+    Field,
+)
+
+from galaxy.exceptions import AdminRequiredException
+from galaxy.managers.histories import HistoryManager
+from galaxy.managers.workflows import WorkflowsManager
+from galaxy.schema.schema import InvocationIndexQueryPayload
+from galaxy.security.idencoding import IdEncodingHelper
+from galaxy.webapps.galaxy.services.base import ServiceBase
+
+
+class InvocationSerializationParams(BaseModel):
+    """Contains common parameters for customizing model serialization."""
+
+    view: Optional[str] = Field(
```

Comment:
Should we have an enum for the available views ?

Reply:
Yes! https://github.com/galaxyproject/galaxy/pull/13875